pubspec.yaml

```
@@ -11,7 +11,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  fluentui_system_icons: ^1.1.83
+  fluentui_system_icons: ^1.1.85
```

```suggestion
  fluentui_system_icons: ^1.1.83
```

Undo upgrading pubspec.yaml here. Preferrably create a new branch after checking out master called chore/upgrade-dependencies to upgrade dependencies. Since the current branch's goal is to create the module card widget and not to upgrade dependencies.